iroha/docs/guides/develop-iroha-module.md

```diff
@@ -0,0 +1,148 @@
+# How-to develop a new Iroha Module
+
+When you need to add some functionality to the Iroha use this guide to develop a new Iroha Module.
+
+## Prerequisites
+
+* [Rust](https://www.rust-lang.org/tools/install)
+* Text Editor or IDE
+
+## Steps
+
+### Create new Rust module inside Iroha crate
+
+Inside `iroha/src/lib.rs` add declaration of your new module.
+For example for `bridge` module we add the following declaration:
+```rust
+#[cfg(feature = "bridge")]
+pub mod bridge;
+```
+
+So for you module `x` you will add `pub mod x;`. You should also place your new module under the 
```

Comment:
"your module", "you need to add" or "you should add" maybe?  
Also, I don't get the line break before the link.